src/_gettsim_tests/test_full_taxes_and_transfers.py

```diff
@@ -92,8 +114,8 @@ def test_data_types(
             elif column_name in year_functions:
                 internal_type = year_functions[column_name].__annotations__["return"]
             else:
-                # TODO (@hmgaudecker): Implement easy way to find out expected type of
-                #     aggregated functions
+                # TODO(@hmgaudecker): Implement easy way to find out
```

Comment:
Seems unrelated, revert? 
```suggestion
                # TODO (@hmgaudecker): Implement easy way to find out expected type of
```